### PR TITLE
verkle: test vector samples & wiring

### PR DESCRIPTION
**Pull request by jsign**
This PR contains ongoing work for Verkle Trees test vectors.

The main goal of the current state of the PR is having some test samples for EIP-6800 and EIP-4762 so we can explore the required changes to the testing library, mainly regarding:
- The t8n tool needs to support a new way of verifying the post-state since we’d now have a single tree instead of account and storage trees.
- The t8n tool needs to support providing the *execution witness* to the testing library so that the `fill` tool can verify that the generated witness for the test is correct.

The required changes for t8n have already been implemented, mainly in [this branch](https://github.com/gballet/go-ethereum/pull/432). 

This PR adds some sample tests that would use the new (to be wired) t8n tool features to bring down to earth what is required in the testing library. I’ll explain better in the comments.

Note that the tests might not be entirely idiomatic and couldn’t be run with `fill` since we’d need first to complete the wiring of this testing library with the new features.

### CODE REVIEW

File: tests/osaka/temp_verkle_helpers.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+def vkt_add_all_headers_present(witness_keys, addr):
+    """
+    Adds to witness_keys that all account headers entries should be present.
+    """
+    witness_keys[vkt_key_header(addr, AccountHeaderEntry.VERSION)] = AccountHeaderEntry.PRESENT
+    witness_keys[vkt_key_header(addr, AccountHeaderEntry.BALANCE)] = AccountHeaderEntry.PRESENT
+    witness_keys[vkt_key_header(addr, AccountHeaderEntry.NONCE)] = AccountHeaderEntry.PRESENT
+    witness_keys[vkt_key_header(addr, AccountHeaderEntry.CODE_HASH)] = AccountHeaderEntry.PRESENT
+    witness_keys[vkt_key_header(addr, AccountHeaderEntry.CODE_SIZE)] = AccountHeaderEntry.PRESENT
```
Comment (jsign) [inline tests/osaka/temp_verkle_helpers.py:88]: Probably we could also make it work with `= {}`

**Comment by acolytec3** [comment]
Trying to build these so I can start trying to run the ethereumjs code against them but I'm not really sure to generate the fixtures.

I've cloned and built geth from the branch referenced in the opening post above and then was trying to build them with something like `fill -v tests/osaka/eip6800_verkle_tree/test_transfer.py --fork=Osaka --evm=../go-ethereum/build/bin/evm` but Osaka doesn't seem to be a recognized fork.  Do I need to add something or cherry-pick a commit from another PR to be able to fill these?

**Comment by jsign** [comment]
> Trying to build these so I can start trying to run the ethereumjs code against them but I'm not really sure to generate the fixtures.
> 
> I've cloned and built geth from the branch referenced in the opening post above and then was trying to build them with something like `fill -v tests/osaka/eip6800_verkle_tree/test_transfer.py --fork=Osaka --evm=../go-ethereum/build/bin/evm` but Osaka doesn't seem to be a recognized fork. Do I need to add something or cherry-pick a commit from another PR to be able to fill these?

Talked with Gabriel, but also writing here. This branch isn't runnable yet. There's TODO work in the testing library to do some wiring with the new t8n features.

**Comment by jsign** [comment]
Work will continue on top of library work here: https://github.com/spencer-tb/execution-spec-tests/pull/41

**Pull request closed without merging**